docs/sql/mysql/create_schema.sql

```diff
@@ -183,33 +183,67 @@ CREATE TABLE da_sms_authorization (
   timestamp_expires    TIMESTAMP DEFAULT CURRENT_TIMESTAMP
 ) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 
+-- Table da_user_credentials stores built-in users for the data adapter
+CREATE TABLE da_user_credentials (
+  user_id               VARCHAR(128) PRIMARY KEY NOT NULL,
+  username              VARCHAR(255) NOT NULL,
+  password_hash         VARCHAR(255) NOT NULL,
+  family_name           VARCHAR(255) NOT NULL,
+  given_name            VARCHAR(255) NOT NULL,
+  organization_id       VARCHAR(64)  NOT NULL,
+  phone_number          VARCHAR(255) NOT NULL
```

Comment:
Why do we need the phone number? Just curious. I would rather expect e-mail address here.